docs/upgrade.html

```diff
@@ -82,6 +82,17 @@ <h4><a id="upgrade_2_4_0" href="#upgrade_2_4_0">Upgrading from 0.8.x, 0.9.x, 0.1
     </li>
 </ol>
 
+<h5><a id="upgrade_241_notable" href="#upgrade_241_notable">Notable changes in 2.4.1</a></h5>
```

Comment:
From what I can see in the file, the `notable` notes for the current release go before the ones for the previous release.